src/main/resources/db/migration/V4.20.0.4__6505-zipdownload-jobs.sql

```diff
@@ -0,0 +1,2 @@
+-- maybe temporary? - work in progress
+CREATE TABLE IF NOT EXISTS CUSTOMZIPSERVICEREQUEST (KEY VARCHAR(63), STORAGELOCATION VARCHAR(255), FILENAME VARCHAR(255), ISSUETIME TIMESTAMP);
```

Comment:
Should we use our usual `@Entity` convention and have JPA create and manage this table? And remove this SQL script?

Reply:
That was on purpose - I wanted this table to live outside what's managed by JPA; and I didn't want to make its entries entities. 
In part because the table is going to be modified externally (the zipper job cleaning after itself and deleting the entries for the jobs it has served). But also to emphasize the custom-ness and hacky-ness of this setup?